prog/dftb+/prg_dftb/parser.F90

```diff
@@ -1333,8 +1333,36 @@ subroutine readDFTBHam(node, ctrl, geo, slako)
     else
       skInterMeth = skEqGridNew
     end if
-    call readSKFiles(skFiles, geo%nSpecies, slako, slako%orb, &
-        &angShells, ctrl%tOrbResolved, skInterMeth, repPoly)
+    call getChild(node, "TruncateSKRange", child, requested=.false.)
+    if (associated(child)) then
+      call warning("Artificially truncating the SK table, this is normally a bad idea!")
+      ! Artificially truncate the SK table
+      call getChildValue(child, "SKMaxDistance", truncationCutOff, -1.0_dp, modifier=modifier,&
+          & child=field)
+      call convertByMul(char(modifier), lengthUnits, field, truncationCutOff)
+      call getChildValue(child, "HardCutOff", tFound, .true.)
+      if (tFound) then
```

Comment:
Could you take a logical variable with more descripting name, rather than reuse an existing one. (Maybe putting the entire `TruncateSKRange` parsing into a separate subroutine would be the cleanest...